python-import/cyclical_imports/yang.py

```diff
@@ -0,0 +1,14 @@
+# yang.py
+
+print("Hello from yang")
+import yin  # noqa
```

Comment:
Is there a reason you're calling `print()` before the `import`? If not, we could swap the order here so that you don't need to suppress the linter. The `# noqa` comment might be confusing...

Reply:
This is part of a cyclical import example, and the `print()` is there to make it clear when each module is imported. So one of the points of the example is that `print()` is before `import`. See https://realpython.com/python-import/#handle-cyclical-imports

One option could be to add some more explanation to `# noqa`:

```
import yin  # noqa  Import should normally be on top of file
```

Another option could be to remove this example from `materials`. I think many of the other examples are useful to offer as "finished" code, but this one doesn't do much useful at the end and the code really only makes sense if you follow along in the tutorial from the first version to this last one.